text/0000-result-expect.md

```diff
@@ -0,0 +1,40 @@
+- Feature Name: `result_expect`
+- Start Date: 2015-05-13
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+Add an `expect` method to the Result type, bounded to `E: Debug`
```

Comment:
A panic caused by an ICE gets caught and then the code goes on to [panic](https://github.com/rust-lang/rust/blob/master/src/librustc_driver/lib.rs#L847). Are you suggesting the latter should be fixed?

Both of you seem to be implying that a panic is not a valid way for a utility to die on a fatal error. Any citation for that?